### DISCUSSION TITLE
How to categorize features in the `features/` directory

### DISCUSSION THREAD

**Discussion by nickyito** [nishiki-tech/nishiki-frontend#78] [General]
Currently, in the `features/`, there are three sub-directories directories. 
- container
- group
- user

I think the features in the `features/` directory should match the ones we decided the other day.

- food management
- profile
- (auth)
- shopping list
- recipe suggestion

Or, we could even divide the "food management" feature into two separate features "groups" and "foods".

**Comment by nickyito** [comment]
@kotaaaa @kanta1207 
May I hear your opinions, please?

**Reply by kotaaaa** [reply]
I prefer this
> Or, we could even divide the "food management" feature into two separate features "groups" and "foods".

How about these directories?
- groups
- foods
- profile
- (auth)
- shopping list
- recipe suggestion

**Reply by nickyito** [reply]
I agree!

**Reply by nickyito** [reply]
Well then, shall we have the `groups`, `foods` and `profile` for now?
Unused ones could be empty with `.gitkeep` for example, `features/profile/.gitkeep`.

**Reply by kotaaaa** [reply]
Anytime is ok for me.

**Reply by nickyito** [reply]
OK, I created an issue (#80) for handling this.

**Comment by nickyito** [comment]
This discussion concluded with the decision to categorize the features listed below:
- groups
- foods
- profile
- auth
- shopping list (Create when needed)
- recipe suggestion (Create when needed)
